flink-runtime/src/main/java/org/apache/flink/runtime/io/network/partition/JobMasterPartitionTrackerImpl.java

```diff
@@ -78,6 +77,22 @@ public void startTrackingPartition(
                 producingTaskExecutorId, resultPartitionId, resultPartitionDeploymentDescriptor);
     }
 
+    @Override
+    public Collection<PartitionTrackerEntry<ResourceID, ResultPartitionDeploymentDescriptor>>
+            stopTrackingPartitionsFor(ResourceID producingTaskExecutorId) {
+        Preconditions.checkNotNull(producingTaskExecutorId);
+
+        // stop tracking partitions which are accommodated on TM
+        Set<ResultPartitionID> rIds =
```

Comment:
I prefer the name to be `resultPartitionIds` just like other places in this class.

Reply:
Sure